xrootd.sh

```
@@ -82,19 +82,17 @@ if [[ "$XROOTD_PYTHON" == "True" ]]; then
 
     pushd lib
     if [ -d ../lib64/python${PYTHON_VER} ]; then
-      ln -s ../lib64/python${PYTHON_VER} python
-    elif [[ -d python${PYTHON_VER} ]]; then
-      ln -s python${PYTHON_VER} python
+      mv ../lib64/python${PYTHON_VER} .
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)